modules/targets/darwin/fonts.nix

```diff
@@ -0,0 +1,27 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  homeDir = config.home.homeDirectory;
+  fontsEnv = pkgs.buildEnv {
+    name = "home-manager-fonts";
+    paths = config.home.packages;
+    pathsToLink = "/share/fonts";
+  };
+  fonts = "${fontsEnv}/share/fonts";
+in {
+  # macOS won't recognize symlinked fonts
+  config.home.activation.copyFonts = hm.dag.entryAfter [ "writeBoundary" ] ''
+    copyFonts() {
+      rm -rf ${homeDir}/Library/Fonts/HomeManager || :
```

Comment:
Usually, we check whether we own the folder (whether it points to the nix store), but it won't be necessary here since we copy fonts to the folder.